script/mediaresilience.js

```diff
@@ -1,16 +1,15 @@
 define(
   'bigscreenplayer/mediaresilience', [
     'bigscreenplayer/models/windowtypes',
-    'bigscreenplayer/models/livesupport',
-    'bigscreenplayer/models/transferformats'
+    'bigscreenplayer/models/playbackstrategy'
   ],
-  function (WindowTypes, LiveSupport, TransferFormats) {
+  function (WindowTypes, PlaybackStrategy) {
     'use strict';
 
     function shouldFailover (remainingUrls, duration, currentTime, liveSupport, windowType, transferFormat) {
       var aboutToEnd = duration && currentTime > duration - 5;
       var shouldStaticFailover = windowType === WindowTypes.STATIC && !aboutToEnd;
-      var shouldLiveFailover = windowType !== WindowTypes.STATIC && (transferFormat === TransferFormats.DASH || liveSupport !== LiveSupport.RESTARTABLE);
+      var shouldLiveFailover = windowType !== WindowTypes.STATIC && window.bigscreenPlayer.playbackStrategy !== PlaybackStrategy.TAL && !window.bigscreenPlayer.disableLiveFailover;
```

Comment:
How come we're introducing yet another configuration variable? `disableLiveFailover`